docs/en/ingest-management/fleet/add-fleet-server-kubernetes.asciidoc

```diff
@@ -0,0 +1,392 @@
+
+Also note that the `service token` required by the {fleet-server} is different from the `enrollment tokens` used by {agent}s to enroll to Fleet.
+====
+
+. In {kib}, open *Fleet → Settings* and ensure the *Elasticsearch output* that will be used by the {fleet-server} policy is correctly configured, paying special attention to:
++
+** The *hosts* field includes a valid URL that will be reachable by the {fleet-server} Pod.
+** If your {es} cluster uses certificates signed by private or intermediate CAs not publicly trusted, ensure to add the trust information in the *Elasticsearch CA trusted fingerprint* field or in the *advanced configuration* section through `ssl.certificate_authorities` setting. For more details refer to https://elastic.co/guide/en/fleet/current/secure-connections.html#_encrypt_traffic_between_elastic_agents_fleet_server_and_elasticsearch[Secure Connections] documentation.
++
+[IMPORTANT]
+====
+This validation step is critical. The {es} host URL and CA information has to be added *in both the {es} output and the environment variables* provided to the {fleet-server} in a later step. It's a common mistake to ignore the output settings believing that the environment variables will prevail, when the environment variables are only used during the bootstrap of the {fleet-server}.
+
+Consider the creation of a dedicated *{es} output* for the {fleet-server} if the URL needed by the {fleet-server} to reach {es} differs from the URL needed by other clients.
+====
+
+. From *Fleet → Agent Policies*, select *Create agent policy* to create a policy for the {fleet-server}:
++
+** Set a *name* to the policy, for example `Fleet Server Policy Kubernetes`
+** Do *not* select the option *collect system logs and metrics*. This option would add the `system integration` to the policy, and considering the {fleet-server} will run as a Kubernetes Pod without any visibility to the Kubernetes node, there won't be any real system to monitor.
+** Select the **output** that the {fleet-server} needs to use to contact {es}. This should be the output that was verified in the previous step.
+** If desired, add an **inactive agent unenrollment timeout** to automatically unenroll and invalidate API Keys after being inactive. This is useful on Kubernetes environments as the {fleet-server} Pods are ephemeral, and after restarts new {agent}s will appear in Fleet UI.
+
+. Open the created policy, and from the *Integrations* tab select *Add integration*:
++
+** Search and select the Fleet Server integration
+** Select *Add Fleet Server* to add the integration to the policy.
++
+At this point you can configure the integration settings per <<fleet-server-scalability>>.
+** When done, select on *Save and continue*. Do not add any {agent} at this stage.
+
+. Open the configured policy, which now includes the Fleet Server integration, and select *Actions* → *Add Fleet Server*. In the next dialog:
++
+* Confirm that the *policy for Fleet Server* is properly selected.
```

Comment:
Just checking: Is `policy for Fleet Server` the string exactly as it appears in the UI?

Reply:
Yes, it appears exactly like that, as part of the sentence Select a policy for the Fleet Server:

![image](https://github.com/user-attachments/assets/a099c89b-620c-4568-8542-a3dddca02ea8)

I'm opened to suggestions here :)

Reply:
Thanks for confirming. It matches the UI so I think your sentence is perfect as is.